garrulous-gobies/BangazonAPI
# Bm active cust

git fetch origin bm-active-cust

# Description
- added bangazon.com to cors origin whitelist in settings.py
- updated README (orders and customers)

# Steps to test solution
- see last bullet point in README customers section
- make sure you have a customer with no open or closed orders
